modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheConfigurationEnrichment.java

```diff
@@ -72,6 +82,16 @@ public String getFieldClassName(String fieldName) {
         return fieldClassNames.get(fieldName);
     }
 
+    /**
+     * Checks that a field presents among of fields for enshrinement.
```

Comment:
Returns {@code true} if this enrichment contains serialized valued for the specified field.

Reply:
Done.